This has compile errors in tests:

```
go test github.com/pinpt/ripsrc/...
# github.com/pinpt/ripsrc/ripsrc/commitmeta/tests [github.com/pinpt/ripsrc/ripsrc/commitmeta/tes
ts.test]
ripsrc/commitmeta/tests/basic_test.go:46:3: unknown field 'Signed' in struct literal of type com
mitmeta.Commit
ripsrc/commitmeta/tests/basic_test.go:62:3: unknown field 'Signed' in struct literal of type com
mitmeta.Commit
ripsrc/commitmeta/tests/merge_basic_test.go:35:3: unknown field 'Signed' in struct literal of ty
pe commitmeta.Commit
ripsrc/commitmeta/tests/merge_basic_test.go:59:3: unknown field 'Signed' in struct literal of ty
pe commitmeta.Commit
ripsrc/commitmeta/tests/merge_basic_test.go:83:3: unknown field 'Signed' in struct literal of ty
pe commitmeta.Commit
ripsrc/commitmeta/tests/merge_basic_test.go:107:3: unknown field 'Signed' in struct literal of t
ype commitmeta.Commit```

--- a/ripsrc/commitmeta/tests/basic_test.go
+++ b/ripsrc/commitmeta/tests/basic_test.go
@@ -43,7 +43,6 @@ func TestBasic(t *testing.T) {
 		Message: "c1",
 		Date:    c1d,
 		Parents: nil,
-		Signed:  false,
 		Ordinal: 1,
 	}
 
@@ -59,7 +58,6 @@ func TestBasic(t *testing.T) {
 		Message: "c2",
 		Date:    c2d,
 		Parents: []string{"b4dadc54e312e976694161c2ac59ab76feb0c40d"},
-		Signed:  false,
 		Ordinal: 2,
 	}
 

--- a/ripsrc/commitmeta/tests/merge_basic_test.go
+++ b/ripsrc/commitmeta/tests/merge_basic_test.go
@@ -32,7 +32,6 @@ func TestMergeBasic(t *testing.T) {
 		},
 		Message: "base",
 		Date:    c1d,
-		Signed:  false,
 		Ordinal: 1,
 	}
 
@@ -56,7 +55,6 @@ func TestMergeBasic(t *testing.T) {
 		Message: "a",
 		Date:    c2d,
 		Parents: []string{"cb78f81991af4120b649c5e2ae18cceba598220a"},
-		Signed:  false,
 		Ordinal: 2,
 	}
 
@@ -80,7 +78,6 @@ func TestMergeBasic(t *testing.T) {
 		Message: "m",
 		Date:    c3d,
 		Parents: []string{"cb78f81991af4120b649c5e2ae18cceba598220a"},
-		Signed:  false,
 		Ordinal: 3,
 	}
 
@@ -104,7 +101,6 @@ func TestMergeBasic(t *testing.T) {
 		Message: "merge",
 		Date:    c4d,
 		Parents: []string{"3219b85f18fad2aa802344a2275bd8288916f4ee", "a08d204ee5913986294000e1280e7ad3484098e3"},
-		Signed:  false,
 		Ordinal: 4,
 	}
 

